pkg/scheduler/framework/framework_test.go

```diff
@@ -1935,3 +1935,387 @@ func TestUpdatePolicySnapshotStatusFrom(t *testing.T) {
+								AffinityScore:       &affinityScore2,
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "same score, different names",
+			bindings: []*fleetv1beta1.ClusterResourceBinding{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: bindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: clusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: altBindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: altClusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: anotherBindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: anotherClusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+			},
+			want: []*fleetv1beta1.ClusterResourceBinding{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: bindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: clusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: anotherBindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: anotherClusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: altBindingName,
+					},
+					Spec: fleetv1beta1.ResourceBindingSpec{
+						TargetCluster: altClusterName,
+						ClusterDecision: fleetv1beta1.ClusterDecision{
+							ClusterScore: &fleetv1beta1.ClusterScore{
+								TopologySpreadScore: &topologySpreadScore1,
+								AffinityScore:       &affinityScore1,
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			sorted := sortByClusterScoreAndName(tc.bindings)
+			if diff := cmp.Diff(sorted, tc.want); diff != "" {
+				t.Errorf("sorted diff (-got, +want): %s", diff)
```

Comment:
```suggestion
				t.Errorf("sortByClusterScoreAndName() mismatch (-got, +want): %s", diff)
```
identify the func?
https://github.com/golang/go/wiki/TestComments#identify-the-function

Reply:
Done.